posts/inside-rust/2020-12-28-cjgillot-and-nadrieril-for-compiler-contributors.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Please welcome cjgillot and Nadrieril to compiler-contributors"
+author: Wesley Wiser
+description: "Please welcome cjgillot and Nadrieril to compiler-contributors"
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Please welcome [@cjgillot] and [@Nadrieril] to the [compiler-contributors] group!
+
+[@cjgillot] has been working to improve the query system used internally in `rustc` which powers incremental compilation.
+Some of their improvements have been to reduce unnecessary work performed during incremental compilation, leading to faster builds while other improvements have made the query system leaner allowing `rustc` to bootstrap faster.
```

Comment:
Lets break line 12 into two sentences: "... leading to faster builds. Other improvements has made ..."